### PR TITLE
klogg: Add shim script

### PR DESCRIPTION
**Pull request by galpressman**
Add shim script to execute klogg.

Signed-off-by: Gal Pressman <galpressman@gmail.com>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

File: Casks/klogg.rb [hunk 1]
```diff
@@ -14,4 +14,14 @@
...
 
   app "klogg.app"
+  # shim script (https://github.com/Homebrew/homebrew-cask/issues/18809)
+  shimscript = "#{staged_path}/klogg.wrapper.sh"
+  binary shimscript, target: "klogg"
+
+  preflight do
+    IO.write shimscript, <<~EOS
+      #!/bin/sh
+      exec '#{appdir}/klogg.app/Contents/MacOS/klogg' "$@" 2>/dev/null
```
Comment (reitermarkus) [inline Casks/klogg.rb:24]: A shim script shouldn't redirect `stderr`.

Comment (GalPressman) [reply Casks/klogg.rb]: Done

**Comment by galpress** [comment]
I updated the PR.

**Comment by GalPressman** [comment]
I updated the PR.

**Comment by GalPressman** [comment]
Kind reminder.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
OK, just noticed a few issues with this cask. Namely, it’s a fork and not named as such. We’ll have to rename it. Furthermore, can you point to any piece of documentation that claims says this shimscript is necessary/wanted? We don’t want to do it for every cask.

**Comment by vitorgalvao** [comment]
@GalPressman Merging this was a bit premature. Can you point to any piece of documentation that says this shimscript is necessary/wanted? We don’t want to do it for every cask.

**Comment by miccal** [comment]
From https://github.com/variar/klogg/blob/master/DOCUMENTATION.md#getting-started:
```
klogg can be started from the command line, optionally passing the file to open as an argument, or via the desktop environment's menu or file association. If no file name is passed, klogg will initially open the last used file.
```
Also see https://github.com/variar/klogg/blob/master/DOCUMENTATION.md#Command-line-options

**Pull request merged**